### DISCUSSION TITLE
Add ability to download WEBP images as PNG / JPEG

### DISCUSSION THREAD

**Discussion by hydro63** [zen-browser/desktop#3712] [Ideas]
I've recently seen some posts of people complaining about how the websites use "WEBP" and according to them WEBP is shit and unusable (in relation to downloading images), and so sites should instead use PNG (which is a really asinine take) + they put the blame of bad support on website authors, rather than graphics programs.

So that got me thinking that this problem should probably be fixed by browsers, rather than website authors. Since browsers can already display WEBP images and they have FFMPEG already built in, correct me if i'm wrong, but it should be pretty painless to do, right?

The user would either be able to specify which format they preferred, or they could choose the format upon downloading the image.

**Comment by rubaboo** [comment]
Workarounds exist. Have a look at https://support.mozilla.org/en-US/questions/1329624.

A little FYI
> very few sites are intentionally designed to use WebP images. YouTube is the only popular one I can think of. What you mostly see is that servers add WebP compression on the fly to JPEG and PNG images because they know Firefox can handle that transparently.
